msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -26,7 +26,7 @@ To integrate location capabilities, you must update the app manifest file and ca
 It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your Teams app.
 
 > [!NOTE] 
-> Currently, Microsoft Teams support for location capabilities is only available for mobile clients.
+> Currently, Microsoft Teams support for location capabilities is available for mobile clients only.
 
 ## Update manifest
```

Comment:
Give a stem statement before the code snippet like:

The following code is an example for devicepermissions and geolocation:

Reply:
It is not an example, itself is the manifest update.